app/models/la_funded_place.rb

```diff
@@ -15,6 +15,6 @@ def to_param
   end
 
   def institution_type
-    'local_authority'
+    'organisation'
```

Comment:
Controversial, buts seems to be mostly used in view code and looks incorrect.

Reply:
:-)